akka-http-tests-java8/src/main/resources/web/calculator.html

```diff
@@ -1,23 +0,0 @@
-<html>
-<body>
-<h1>Calculator</h1>
-
-<h2>Add</h2>
-<form action="add">
-    <label for="x">x:</label><input id="x" name="x" type="text" value="42"/>
-    <label for="y">y:</label><input id="y" name="y" type="text" value="23"/>
-    <input type="submit" />
-</form>
-
-<h2>Subtract</h2>
-<form action="subtract">
-    <label for="x">x:</label><input id="x" name="x" type="text" value="42"/>
-    <label for="y">y:</label><input id="y" name="y" type="text" value="23"/>
-    <input type="submit" />
-</form>
-
-<h3>Multiply</h3>
-<a href="/multiply/42/23">/multiply/42/23</a>
-
-</body>
-</html>
```

Comment:
It's already present in `akka-http-tests/src/main/resources/web/calculator.html`